tests/pytest/pytest_sample/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  pytest.sample:
+    platform_allow: qemu_x86 qemu_x86_64
```

Comment:
This causes the same test to be run 2x. As I understand, the platform chosen has no impact on the actual test. It just serves as a stub so sth is build and handling lines can trigger calling pytest. I would say that there should be just a single "native_posix" if we accept this workflow.

Reply:
updated in #33227